### ISSUE TITLE
[Question] How to define epoch & loss threshold?

### ISSUE THREAD

**Issue by hey24sheep** [axa-group/nlp.js#1348]
Hi, first of all thanks for this great library. 

It would be great if there is a documentation for the corpus + config files especially. Like what all keys and values are supported and what all they mean.

I would really appreciate if someone could tell me 
- How can we increase our epochs to maybe improve accuracy? I want to fiddle with the hyperparams. 
- How to set the minimum acceptable loss before it can stop training
- Also can we define our own maximum perceptrons?

Thanks

**Comment by hey24sheep** [comment]
Any update or guide would be helpful, thanks
